Area51/Sources/TableViewDataSource.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  TableViewDataSource.swift
+//  Area51
```

Comment:
File creation docstring is not needed. I will be longing against these in near future.